docs/en/writing-running-appium/caps.md

```diff
@@ -137,8 +137,8 @@ Appium tries to guess the version of the WebView based on the version of the cor
 |`buildToolsVersion`| Specify the Android `build-tools` version to be something different than the default, which is to use the most recent version. It is helpful to use a non-default version if your environment uses alpha/beta build tools. | e.g. `'28.0.3'` |
 |`androidNaturalOrientation`| Allow for correct handling of orientation on landscape-oriented devices. Set to `true` to basically flip the meaning of `PORTRAIT` and `LANDSCAPE`. Defaults to `false` | `true`, `false` |
 | `enforceAppInstall` | By default application installation is skipped if newer or the same version of this app is already present on the device under test. Setting this option to `true` will enforce Appium to always install the current application build independently of the currently installed version of it. Defaults to `false`. | `true` , `false` |
-| `ignoreHiddenApiPolicyError` | Ignores `Security exception: Permission denial` alert to proceed the create session process since Appium 1.18.0+.. The error happens when Appium try to relax [hidden API policy](https://developer.android.com/distribute/best-practices/develop/restrictions-non-sdk-interfaces#how_can_i_enable_access_to_non-sdk_interfaces) for Appium feature. But some devices restrict the permission. Defaults to `false`. | `true`, `false` |
-| `mockLocationApp` | Set an arbitrary mock location provider as bundle id for a real device since Appium 1.18.0+. Defaults to Appium's settings app. If the value is `null`, Appium will skip the location provider setup. | `null`, `io.appium.settings`, `example.your.app` |
+| `ignoreHiddenApiPolicyError` | Ignores `Security exception: Permission denial` alert and allows to proceed the session creation process since Appium 1.18.0+. The error happens when Appium tries to relax [hidden API policy](https://developer.android.com/distribute/best-practices/develop/restrictions-non-sdk-interfaces#how_can_i_enable_access_to_non-sdk_interfaces) for Appium feature, although some devices with a customized firmware deny such requests. Defaults to `false`. | `true`, `false` |
```

Comment:
...to proceed to the session creation.. or ...to continue the session creation...

Reply:
`for Appium feature` part is not needed